pkg/v1/tkg/tkgconfigpaths/filepaths.go

```diff
@@ -124,3 +124,12 @@ func GetRegistryTrustedCACertFileForWindows() (string, error) {
 	}
 	return path.Join(home, constants.TKGRegistryTrustedRootCAFileForWindows), nil
 }
+
+// GetLogDirectory returns the directory path where log files should be stored by default.
+func (c *client) GetLogDirectory() (string, error) {
+	tkgDir, err := c.GetTKGDirectory()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(tkgDir, constants.LogFolderName), nil
```

Comment:
Q: Should we consider env var`XDG_STATE_HOME` if set, else use the default (base) dir `$HOME/.local/state` ?

Reply:
I think at least for now it should be the same path as `GetTKGDirectory` provides. We can decide later if we would like to customize things, but at least initially I think it makes sense to keep everything contained to the same directory tree.

That's my take on things, but maybe a good topic for discussion. I'm certainly open to hearing what others think.